sdk/storage/storage-blob/karma.conf.js

```diff
@@ -39,7 +40,12 @@ module.exports = function(config) {
 
     // preprocess matching files before serving them to the browser
     // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
-    preprocessors: {
+    preprocessors: testingAllBrowsers? {
+      "**/*.js": ["env"],
+      // IMPORTANT: COMMENT following line if you want to debug in your browsers!!
+      // Preprocess source file to calculate code coverage, however this will make source file unreadable
+      "recordings/browsers/**/*.json": ["json"]
+    } : {
       "**/*.js": ["env"],
```

Comment:
This means that the recordings are loaded only when we test all the browsers?

Reply:
Old code

```js
    // preprocess matching files before serving them to the browser
    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
    preprocessors: {
      "**/*.js": ["env"],
      // IMPORTANT: COMMENT following line if you want to debug in your browsers!!
      // Preprocess source file to calculate code coverage, however this will make source file unreadable
      "dist-test/index.browser.js": ["coverage"],
      "recordings/browsers/**/*.json": ["json"]
    },
```

Also, this line.. `"dist-test/index.browser.js": ["coverage"],` for code coverage by the browser tests is removed irrespective of `testingAllBrowsers` being true or false?

Reply:
I only remove coverage line from `expresion1` for the `true` case in the `condition? expression1 : expression2` ternary operation.  Looks like I do need to remove the comments as well.

Reply:
(could be that the github unified diff makes it confusing)